packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java

```diff
@@ -558,7 +560,8 @@ public ResponseEntity<TaskResponse> createCompareModelsTask(
 						content = @Content)
 			})
 	public ResponseEntity<TaskResponse> createGenerateResponseTask(
-			@RequestParam(name = "mode", required = false, defaultValue = "ASYNC") final TaskMode mode,
+			@RequestParam(name = "mode", required = false, defaultValue = "SYNC") final TaskMode mode,
```

Comment:
Why are we changing the default run mode to SYNC?

Reply:
I thought this endpoint might be used in sync mode most likely if that's easier from the front end. But I can change the default back to ASYNC if that's preferable.

Reply:
I think most likely async, otherwise there'd a lot of waiting around to get the summary id reference. But should be fine for now.